packages/node/src/indexer/api.service.test.ts

```diff
@@ -252,4 +251,40 @@ describe('ApiService', () => {
     expect(call?.section).not.toBe('utility');
     expect(registry).not.toBe(registry2);
   }, 30000);
+
+  it('support historic api rpc', async () => {
+    const apiService = await prepareApiService();
+    const api = apiService.getApi();
+    const patchedApi = await apiService.getPatchedApi();
+
+    const blockhash = await api.rpc.chain.getBlockHash(4401242);
+    await apiService.setBlockhash(blockhash, true);
+
+    const patchedBlock = await patchedApi.rpc.chain.getBlock();
```

Comment:
can you improve the naming, `patchedBlock` feels like the block is patched, but actually it means the block is get from patchedApi. Even `b1`, `b2`,... is better

Reply:
Fixed